buildbuddy.yaml

```diff
@@ -10,17 +10,6 @@ actions:
           - "*"
     bazel_commands:
       - test //... --config=linux-workflows --config=race --test_tag_filters=-performance,-webdriver,-docker,-bare
-  - name: Test with static
```

Comment:
Could you file a bug to re-enable this workflow with a reference to the upstream issues, so that we don't lose track?

Reply:
Tracking it here https://github.com/buildbuddy-io/buildbuddy-internal/issues/3245